.circleci/config.yml

```diff
@@ -547,15 +547,17 @@ jobs:
           path: /home/circleci/project/
       - attach_workspace:
           at: /home/circleci
-      - setup_remote_docker
+      - setup_remote_docker:
+          version: 18.09.3
```

Comment:
Out of interest, why do we need to pin the version here?

Reply:
Uses an ancient version if we don't - specifically, we need a newer engine so we can use env vars in combination with `COPY`